### PR TITLE
feat(): Forked Class name contracts so we can have one for Base and one for MSFT

### PR DESCRIPTION
**Pull request by eljefe223**
Issue: #250 

<body>

<footer>
 

For [details on formatting](https://github.com/Microsoft/fast-dna/wiki/pull-request-guidance) pull requests.

### CODE REVIEW

File: packages/fast-components-class-name-contracts-msft/README.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# FAST components class name contracts
```
Comment (chrisdholt) [inline packages/fast-components-class-name-contracts-msft/README.md]: nit: MSFT

File: packages/fast-components-class-name-contracts-msft/src/managed-classes.ts [hunk 2]
```diff
@@ -0,0 +1 @@
+export * from "@microsoft/fast-components-class-name-contracts-base";
```
Comment (chrisdholt) [inline packages/fast-components-class-name-contracts-msft/src/managed-classes.ts]: Are we sure we don't need to import this?

Comment (eljefe223) [reply packages/fast-components-class-name-contracts-msft/src/managed-classes.ts]: @nicholasrice I don't think we do and if I try I get 'Circular definition of import' error

Comment (nicholasrice) [reply packages/fast-components-class-name-contracts-msft/src/managed-classes.ts]: You do not need to import it

**Pull request merged**